qlib/contrib/data/handler.py

```diff
@@ -426,6 +426,314 @@ def use(x):
 
         return fields, names
 
+class Alpha158EtfFeat(DataHandlerLP):
```

Comment:
shorten the class by inheriting from Alpha158 class

Reply:
Done!